build/Dockerfile

```diff
@@ -2,8 +2,6 @@ ARG ARCH=amd64
 
 FROM quay.io/sustainable_computing_io/kepler_builder:ubi-8.6-bcc-0.24-go1.18 as builder
 
-
-ARG MAKE_TARGET=cross-build-linux-$ARCH
```

Comment:
@rootfs, I don't like to specific a build command, as it makes our make file too big and different branches there. for example we can use goos but there different parameters as goos and os there... I will go to sleep for today, and take this pr later.
if you have time, could you please check with other maintainers for their feed back on how to samplify ?